sktime/transformations/series/kalman_filter.py

```diff
@@ -1,15 +1,16 @@
 # copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
 """Kalman Filter Transformers.
 
-Series based transformers, based on Kalman Filter algorithm. Contains Base class and two
-transformers which are each Adapters for external packages pykalman and FilterPy.
+Series based transformers, based on Kalman Filter algorithm. Contains Base class and 3
```

Comment:
this is a minimal request, but perhaps better for maintaining the code.

I would recommend to turn `kalman_filter` into a folder, and move the current py file in there and make it private. The `__init__.py` of the folder module imports from two files.

This is to avoid too many estimators in a file, which is hard to maintain. The two in the current file have common dependencies, so it is an edge case.

Reply:
Done in https://github.com/sktime/sktime/pull/8030/commits/00434a0dba8da8d4d636e6962a43efbfb897ef7c